unimod_mapper/unimod_mapper.py

```diff
@@ -277,6 +290,24 @@ def name2first_id(self, unimod_name):
         index = min(self.mapper.get(unimod_name, None))
         return self._data_list_2_value(index, "unimodID")
 
+    def name2nl_list(self, unimod_name):
```

Comment:
Spelled out and also added the id2neutral_loss_list function. 

The reason why I had only one is because at the point of neutral loss retrieval (independent if you come from ID or name) you have them both, so one of them is sufficient enough. But for consistency, I would agree that having both will not harm :)